schedule/install/agama_install.yaml

```
@@ -1,8 +1,19 @@
 name:           leap16_agama_install
 description:    >
     This is prepare install opensuse-16.0 agama installer
+conditional_schedule:
+  bootloader:
+    ARCH:
+      aarch64:
+        - installation/bootloader
+      ppc64le:
+        - installation/bootloader
+      s390x:
+        - installation/bootloader_s390
+      x86_64:
+        - installation/bootloader_uefi
```

```suggestion
        - installation/bootloader
```
bootloader_uefi doesn't have the necessary code to boot agama installer: https://openqa.opensuse.org/tests/5272382 (bootloader_start is a wapper, and calls bootloader_uefi)

---

That's mostly mismatching needles (and code missing to press down until on-inst, just like in the upgrade scenarios)